streams-bootstrap-cli/src/test/java/com/bakdata/kafka/CliTest.java

```diff
@@ -287,28 +287,28 @@ public void run() {
                     "--brokers", "brokers",
                     "--schema-registry-url", "schema-registry",
                     "--input-topics", "input1,input2",
-                    "--extra-input-topics", "role1=input3,role2=input4;input5",
+                    "--named-input-topics", "role1=input3,role2=input4;input5",
```

Comment:
Do we want to change `role` to `name`?

Reply:
Im the example we can. I. The Java API I was unsure because it can be kinda confused with topic name and every topic obviously has a name. Maybe we find better terms?

Reply:
some suggestions:
- alias
- identifier
- tag
- label
- stream

Reply:
I think key could also become very confusing given their meaning in Kafka. I like label and tag, identifier is also ok but a bit too long :smile:

Reply:
so labeled-input-topics etc?